library/camel-kamelets/src/main/resources/kamelets/slack-source.kamelet.yaml

```diff
@@ -37,6 +37,12 @@ spec:
       - token
     type: object
     properties:
+      serverUrl:
```

Comment:
Do we need to set it as required?

Reply:
no it should be optional

Reply:
in most cases this will connect to the production Slack API servers (which is also the default in the Camel component). Only tests and maybe a company proxy may use this setting to use a different server URL endpoint